common_module.py

```diff
@@ -227,7 +226,8 @@ def process_line(self, re_search, line, re_match_positions):
                   help="This may be used in situations where node names are not a pure arithmetic seq (eg. rocks clusters)")
 parser.add_option("-A", "--anonymize", action="store_true", dest="ANONYMIZE", default=False,
                   help="Masks unix account names and workernode names for security reasons (sending bug reports etc.)")
-parser.add_option("-b", "--batchSystem", action="store", type="string", dest="BATCH_SYSTEM", default=None)
+parser.add_option("-b", "--batchSystem", action="store", dest="BATCH_SYSTEM",
+                  choices=['SGE', 'OAR', 'PBS', 'sge', 'oar', 'pbs', 'auto'], default=None)
```

Comment:
users should be limited to what they can type (fail early!), on the other hand these must not be hardwired in the future! (added as issue #101)